spi_nand_flash/src/nand.c

```diff
@@ -84,6 +84,24 @@ static esp_err_t detect_chip(spi_nand_flash_device_t *dev)
     }
 }
 
+static esp_err_t enable_quad_io_mode(spi_nand_flash_device_t *dev)
+{
+    uint8_t io_config;
+    esp_err_t ret = spi_nand_read_register(dev, REG_CONFIG, &io_config);
+    if (ret != ESP_OK) {
+        return ret;
+    }
+
+    io_config |= QEbit;
+    ESP_LOGD(TAG, "%s: quad config register value: %d\n", __func__, io_config);
```

Comment:
```suggestion
    ESP_LOGD(TAG, "%s: quad config register value: %d", __func__, io_config);
```

or even `%x`, register values are more readable in hex?